### PR TITLE
Gui Accessibility

### PR DESCRIPTION
**Pull request by jmdaweb**
### Purpose

This pull request fixes some gui accessibility problems. Some html tags have been closed, and alt text has been added to some images. Also, aria-required attribute has been added to some input form fields.

### Testing

You can compare the old and new html files. You will see less errors and warnings. You can use the W3C validator at http://validator.w3.org/. Also, a screen reader can be used to browse the application, for example NVDA (https://nvaccess.org).

### Authorship

Jose Manuel Delicado

### CODE REVIEW

**Comment by wweich** [comment]
In HTML5 `type="text/javascript"`isn't needed anymore.
Afaik, `required` is valid, without `=""`or `="required"` in HTML5.

File: gui/default/syncthing/device/globalChangesModalView.html [hunk 1]
```diff
@@ -2,22 +2,26 @@
 <modal id="globalChanges" status="default" icon="{{'history'}}" heading="{{'Global Changes' | translate}}" large="yes" closeable="yes">
     <div class="modal-body">
         <table>
-          <tr>
+<thead>          
+<tr>
```
Comment (AudriusButkevicius) [inline gui/default/syncthing/device/globalChangesModalView.html]: Indentation

File: gui/default/index.html [hunk 2]
```diff
@@ -9,32 +9,32 @@
...
   <div ng-if="version.isDevelopmentVersion" ng-include="'syncthing/development/logbar.html'"></div>
   <!-- Top bar -->
 
   <nav class="navbar navbar-top navbar-default" role="navigation">
     <div class="container">
       <span class="navbar-brand" aria-hidden="true">
-        <img class="logo hidden-xs" src="assets/img/logo-horizontal.svg" height="32" width="117"/>
-        <img class="logo hidden visible-xs" src="assets/img/favicon-default.png" height="32"/>
+        <img class="logo hidden-xs" src="assets/img/logo-horizontal.svg" height="32" width="117" alt=""/>
+        <img class="logo hidden visible-xs" src="assets/img/favicon-default.png" height="32" alt=""/>
```
Comment (AudriusButkevicius) [inline gui/default/index.html]: Should fill the alt explaining what it is.

File: gui/default/syncthing/device/idqrModalView.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <modal id="idqr" status="info" icon="qrcode" heading="{{'Device Identification' | translate}} - {{deviceName(currentDevice)}}" large="yes" closeable="yes">
   <div class="modal-body">
     <div class="well well-sm text-monospace text-center" select-on-click>{{currentDevice.deviceID}}</div>
-    <img ng-if="currentDevice.deviceID" class="center-block img-thumbnail" ng-src="qr/?text={{currentDevice.deviceID}}"/>
+    <img ng-if="currentDevice.deviceID" class="center-block img-thumbnail" ng-src="qr/?text={{currentDevice.deviceID}}" alt="qr"/>
```
Comment (AudriusButkevicius) [inline gui/default/syncthing/device/idqrModalView.html]: Qr doesn't explain much. Qr code maybe?

**Comment by jmdaweb** [comment]
Sorry, I didn’t take into account the indentation, I’m fixing it. I’ll change the qr to qr code too. In the other 2 images, alt attribute is required, but can be empty. The aria-hidden attribute in the container makes a screen reader ignore the content.
The purpose of required=”” is increasing the compatibility with other html validators.
Regards.

**Comment by uok** [comment]
my personal preference is `alt="" />` instead of `alt=""/>`

**Comment by calmh** [comment]
These are all good fixes, thank you, lgtm. I'll sort out the author thing tomorrow so it can be merged.

**Comment by calmh** [comment]
@st-jenkins retest this please

**Comment by calmh** [comment]
@st-review lgtm

**Comment by st-review** [comment]
@calmh: Noted! Need another LGTM or explicit merge command.

**Comment by calmh** [comment]
(note, needs commit message fixup when merging)

**Comment by calmh** [comment]
@st-jenkins retest this please

**Comment by calmh** [comment]
@st-review merge it

gui: HTML accessibility updates

**Comment by st-review** [comment]
:ok_hand: Merged as 4405117bea28a653d427207a60bac3ca297c9484. Thanks, @jmdaweb!

**Pull request closed without merging**